editor/editor_node.cpp

```diff
@@ -5024,6 +5024,35 @@ bool EditorNode::immediate_confirmation_dialog(const String &p_text, const Strin
 	return singleton->immediate_dialog_confirmed;
 }
 
+Color EditorNode::peek_cfg_theme_color() {
+	// Default black(0,0,0,1)
+	Color result;
```

Comment:
I wouldn't document what the default color is, if this ever changes it will make the comment obsolete.
Moreover, you never assign `result` so it's not relevant to define here, you can just do `return Color();` in the fallback case.